cranelift/codegen/meta/src/isa/x86/opcodes.rs

```diff
@@ -354,6 +354,10 @@ pub static PADDUSB: [u8; 3] = [0x66, 0x0f, 0xdc];
 /// Add packed unsigned word integers from xmm2/m128 and xmm1 saturate the results (SSE).
 pub static PADDUSW: [u8; 3] = [0x66, 0x0f, 0xdd];
 
+/// Concatenate destination and source operands, extract byte-aligned result shifted to the right by
+/// the constant value in imm8 into xmm1 (SSSE3).
```

Comment:
Please specify whether the shift is by `imm8` bits or `imm8` bytes.  I think it's the latter but I don't exactly remember.